provider/mock/mock.go

```diff
@@ -257,15 +257,15 @@ func (p *MockV0Provider) DeletePod(ctx context.Context, pod *v1.Pod) (err error)
 	now := metav1.Now()
 	delete(p.pods, key)
 	pod.Status.Phase = v1.PodSucceeded
-	pod.Status.Reason = "MockProviderPodDeleted"
+	pod.Status.Reason = "ProviderPodDeleted"
 
 	for idx := range pod.Status.ContainerStatuses {
 		pod.Status.ContainerStatuses[idx].Ready = false
 		pod.Status.ContainerStatuses[idx].State = v1.ContainerState{
 			Terminated: &v1.ContainerStateTerminated{
 				Message:    "Mock provider terminated container upon deletion",
 				FinishedAt: now,
-				Reason:     "MockProviderPodContainerDeleted",
+				Reason:     "ProviderPodContainerDeleted",
```

Comment:
This provider is replicated between the node-cli repo, and the VK repo, why?

Reply:
I think the intention was to have mock live here. It's been so long at this point, though.